system-reinstall-bootc/src/podman.rs

```diff
@@ -41,6 +41,9 @@ pub(crate) fn reinstall_command(image: &str, ssh_key_file: &str) -> Command {
         // this binary. Since this is no longer an "arcane" bootc command, we can safely avoid this
         // timed warning prompt. TODO: Discuss in https://github.com/containers/bootc/discussions/1060
         "--acknowledge-destructive",
+        // The image is always pulled into containers-storage first, so let's use it here to avoid
+        // requiring registry keys to be baked into the image.
```

Comment:
We want `--skip-fetch-check` here to avoid that.

And honestly I'm OK to just drop that check by default.